packages/kbn-es-query/src/es_query/es_aggregate_query.ts

```diff
@@ -66,3 +66,8 @@ export function getIndexPatternFromESQLQuery(esql?: string): string {
   }
   return '';
 }
+
+export function cleanupESQLQuery(esql?: string): string {
```

Comment:
Would be great to describe the purpose of the clean up. Or at least change the function name to something like `cleanupESQLQueryForLensSuggestions`

Reply:
Fixed with fc4c37f001bc1e65e9ba50aa698c6e952be49529